.github/workflows/release.yml

```diff
@@ -36,25 +34,37 @@ jobs:
           node-version: 18
 
       - run: npm install
-
       - run: npm run build
-
       - run: npm run test
         if: ${{ inputs.skip-test == false }}
+      - name: Upload release artifacts
+        id: upload-release
+        uses: actions/upload-artifact@v4
+        with:
+          name: release
+          path: lib
 
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v2
+        uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: ${{ secrets.AWS_CODEARTIFACT_ROLE }}
           aws-region: us-west-2
-      - name: Login and configure codeartifact
+      - name: Run CodeBuild
+        uses: aws-actions/aws-codebuild-run-build@v1
+        with:
+          project-name: Importer
+          disable-source-override: true
+          hide-cloudwatch-logs: true
+          env-vars-for-codebuild: |
+            GITHUB_ARTIFACT_ID,
+            GITHUB_REPOSITORY_NAME,
+            GITHUB_COMMIT_MESSAGE,
+            GITHUB_BRANCH_NAME,
+            GITHUB_TOKEN
```

Comment:
For how long is this one valid? Are we sure that it is always work for the duration of our builds and imports?

Reply:
The token is valid for the duration of the action.

Reply:
This is fine I think. Restarting the whole Github action works.